cmd/libs/go2idl/client-gen/output/deployment.go

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
According to #15730, how about output to pkg/client/clientset/unversioned? If we output the files there, then we can merge the output now, otherwise we need to wait until the generated code is perfect.